src/components/Switch/SwitchTypes.ts

```diff
@@ -12,5 +12,6 @@ export interface SwitchProps
   > {
   error?: boolean;
   multiline?: boolean;
+  textOnLeft?: boolean;
```

Comment:
nit: personal preference, I'd prefer a "placement" position such as
```suggestion
  textPosition?: "left" | "right", // defaults to "left" 
```
or 
```suggestion
  childrenPlacement?: "left" | "right", // defaults to "left" 
```
or you could use `switchPosition`/`switchPlacement`/ect.

This is more intuitive to me, and allows us to extend the logic to include above/below/ect. in the future if we need to. 

But again, personal preference, so feel free to ignore.

Reply:
changed to textPlacement